KHLY2700/KHLY2700-IOC-01App/Db/keithley2700.db

```diff
@@ -0,0 +1,733 @@
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(bo, "$(P)BUFFER:CLEAR:SP")
+{
+	field(DESC, "Clear readings from buffer")
+	field(DTYP, "stream")
+	field(ZNAM, "")
+	field(ONAM, "")
+	field(OUT, "@devKeithley_2700.proto clearBuffer $(PORT)")    
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:CLEAR:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)BUFFER:CLEAR:SP","$(P)BUFFER:CLEAR")
+
+record(bo, "$(P)SYSTEM:CLEAR:SP")
+{
+	field(DESC, "Clear error message from queue")
+	field(DTYP, "stream")
+	field(ZNAM, "")
+	field(ONAM, "")
+	field(OUT, "@devKeithley_2700.proto clearSystemErrors $(PORT)")  
+
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:SYSTEM:CLEAR:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)SYSTEM:CLEAR:SP","$(P)SYSTEM:CLEAR")
+
+record(mbbo, "$(P)MEASUREMENT:SP") {
+	field(DESC, "Set measurement function")
+	field(DTYP, "stream")
+	field(ZRST, "VOLT")
+    field(ONST, "VOLT:AC")
+    field(TWST, "CURR")
+    field(THST, "CURR:AC")
+    field(FRST, "RES")
+    field(FVST, "FRES")
+    field(SXST, "CONT")
+    field(SVST, "FREQ")
+    field(EIST, "PER")
+	field(OUT, "@devKeithley_2700.proto setMeasurement($(P)CH) $(PORT)")  
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:MEASUREMENT:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)MEASUREMENT:SP","$(P)MEASUREMENT")
+
+record(mbbo, "$(P)BUFFER:SOURCE:SP") 
+{
+    field(DESC, "Get buffer feed source")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(OUT,  "@devKeithley_2700.proto setBufferFeedSource $(PORT)")
+    field(ZRST, "SENS")
+    field(ONST, "CALC")
+    field(TWST, "NONE")
+	info(ARCHIVE, "VAL")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:SOURCE:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)BUFFER:SOURCE:SP", "$(P)BUFFER:SOURCE")
+
+record(bo, "$(P)BUFFER:CLEAR:AUTO:SP") 
+{
+    field(DESC, "Set Buffer State")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(OUT,  "@devKeithley_2700.proto setBufferAutoClearStatus $(PORT)")
+    field(ZNAM, "ON")
+    field(ONAM, "OFF")
+	info(ARCHIVE, "VAL")
+    info(INTEREST, "HIGH")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:CLEAR:AUTO:SP")
+    field(SDIS, "$(P)DISABLE")
+}
+alias("$(P)BUFFER:CLEAR:AUTO:SP","$(P)BUFFER:CLEAR:AUTO")
+
+record(ai, "$(P)BUFFER:NEXT") 
+{
+    field(DESC, "Next Buffer reading location")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getNextBufferReadingLoc $(PORT)")
+	info(ARCHIVE, "VAL")
+    info(INTEREST, "HIGH")
+	field(EGU, "")
+	field(FLNK, "$(P)COUNT:CALC")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER:NEXT")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ao, "$(P)COUNT")
+{
+	field(DESC, "Number of buffer readings to return")
+	info(INTEREST, "LOW")
+	field(EGU, "")
+}
+
+record(ao, "$(P)CURRENT")
+{
+	field(DESC, "Current buffer read location")
+	info(INTEREST, "low")
```

Comment:
Should this not be "LOW"? I thought interests had to be capitalised

Reply:
Capitalised